src/globus_cli/commands/endpoint/user_credential/delete.py

```diff
@@ -7,7 +7,7 @@
 from ._common import user_credential_id_arg
 
 
-@command("delete", short_help="Delete a specific User Credential on an Endpoint")
+@command("delete", short_help="Delete a specific User Credential on an Endpoint.")
```

Comment:
In some places, the `short_help` parameter is deleted rather than updated. Should this be removed, too?

Reply:
In this case, the `short_help` is very slightly different from the first line of the docstring, which is why it's kept separate here.

I've tried to only make more extensive changes when it has 0 effect on the text or is strictly necessary. (But, of course, when working through a large changeset like this one it's hard to stay disciplined.)